src/main/scala/Garage/DBConnection.scala

```diff
@@ -0,0 +1,15 @@
+package Garage
+
+import org.mongodb.scala._
+object DBConnection extends App {
+  val mongoClient: MongoClient = MongoClient("mongodb://localhost")
```

Comment:
you should have a separate method for getting the client as you might not all the time use a locally hosted mongo db